src/main/java/uk/gov/hmcts/reform/professionalapi/controller/SuperController.java

```diff
@@ -112,7 +112,6 @@
     protected ResponseEntity<OrganisationResponse>  createOrganisationFrom(OrganisationCreationRequest organisationCreationRequest) {
 
         organisationCreationRequestValidator.validate(organisationCreationRequest);
-        OrganisationCreationRequestValidator.validateJurisdictions(organisationCreationRequest.getSuperUser().getJurisdictions(), prdEnumService.getPrdEnumByEnumType(jurisdictionIds));
```

Comment:
I have removed jurisdiction validation here because the confluence says: "PRD will remove the validation of the Jurisdiction IDs of users that get added via ExUI - this does include the Jurisdiction IDs of the super-user as well"